zuul.d/projects.yaml

```diff
@@ -202,6 +220,25 @@
         - docker-machine-functional-devstack-mitaka:
             branches: master
 
+- project:
+    name: gophercloud/gophercloud
+    periodic-8/20:
+      jobs:
+        - gophercloud-acceptance-test-stein:
+            branches: master
+        - gophercloud-acceptance-test-rocky:
+            branches: master
+        - gophercloud-acceptance-test-queens:
+            branches: master
+        - gophercloud-acceptance-test-pike:
+            branches: master
+        - gophercloud-acceptance-test-ocata:
+            branches: master
+        - gophercloud-acceptance-test-newton:
+            branches: master
+        - gophercloud-acceptance-test-mitaka:
+            branches: master
```

Comment:
Please merge above gophercloud jobs (see line 122) into this gophercloud project

Reply:
Done. And I remove the ut test job.